pyproject.toml

```diff
@@ -27,13 +27,13 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-pyyaml = "^5.1"
-moto = "^1.3"
-boto3 = "^1.9"
+python = "^3.7"
```

Comment:
With upgraded version of `python`, we can also remove https://github.com/whisller/pytest-serverless/blob/master/pyproject.toml#L17-L18 

After this is merged I will create new release of `1.0.0`.